### ISSUE TITLE
Use API error message with "other" error handling

### ISSUE THREAD

**Issue by gordonsilvera** [bogdanteodoru/py3cw#13]
The [error msg for miscellanous errors](https://github.com/bogdanteodoru/py3cw/blob/master/py3cw/request.py#L132) is a bit vague. In my case, I put the wrong pair format (eg BTC-USDT), but couldn't see that was the error.

You can provide more detailed information with something such as:
```
        except Exception:
            return {'error': True, 'msg': 'Other error occurred: {} {}.'.format(
                response_json.get('error'), response_json.get('error_description'))}, None

```

Love the library otherwise!

**Comment by bogdanteodoru** [comment]
Hi @gordonsilvera, the error you see there is the error I get from the 3commas API. Is not something that I output myself. So if the error is vague from them, I just pass it further. 

But you can do a PR with that code I can merge it and build a new version :)

**Comment by bogdanteodoru** [comment]
Fixed in version 0.0.21

**Comment by gordonsilvera** [comment]
Thanks!
